pkg/ruler/ruler.go

```diff
@@ -288,6 +290,16 @@ func newRuler(cfg Config, manager MultiTenantManager, reg prometheus.Registerer,
 			Name: "cortex_ruler_sync_rules_total",
 			Help: "Total number of times the ruler sync operation triggered.",
 		}, []string{"reason"}),
+
+		ruleGroupStoreLoadDuration: promauto.With(reg).NewGauge(prometheus.GaugeOpts{
+			Name: "cortex_ruler_rule_group_store_load_duration",
```

Comment:
Can we just simplify it to `cortex_ruler_rule_group_load_duration`?

Also, let's add suffix for the metric like `_seconds`.https://prometheus.io/docs/practices/naming/#metric-names